### DISCUSSION TITLE
Photosphere/Panorama Image Viewer?

### DISCUSSION THREAD

**Discussion by jfuehner** [visgl/deck.gl#9080] [Q&A]
Was curious if deck could be used to implement a photosphere/panorama image viewing capability similar to [A-Frame](https://aframe.io/examples/showcase/sky/) or [Pannellum](https://pchen66.github.io/Panolens/#Example)? Perhaps using a combination of OrbitView or FirstPersonView and the BitmapLayer?

**Comment by Pessimistress** [comment]
Like https://deck.gl/examples/first-person-view ?

**Comment by jfuehner** [comment]
@Pessimistress thanks for pointing me to that example. Super helpful! 

I noticed the controllers scrollZoom and doubleClickZoom properties were set to false in the first-person-view example. Interestingly, when using a image (i.e., JPG) as the SimpleMeshLayer's texture if I set the scrollZoom property to true then I try to articulate left/right the image disappears... However, if I scroll zoom in/out first then articulate left/right the image does not disappear... 


Is this a known issue?
